### DISCUSSION TITLE
mpv lazy 全屏偶尔触发卡死

### DISCUSSION THREAD

**Discussion by heisiheisi** [hooke007/mpv_PlayKit#335] [mpv 通用]
mpv版本:mpv-lazy-2023V5 标准版, 没有更改设置
系统版本: win10 x64
显卡: amd rx6600

用mpv-lazy播放视频, 从窗口模式进入全屏模式后, 有几率触发卡死 (鼠标能动,但无法跳出全屏, 无法进行任何操作,也无法结束进程,只能硬重启)
用potplayer则一直没问题. 请问大家有遇到吗?  请教解决办法

**Comment by hooke007** [comment]
信息太少无法判断。
用纯净模式去测，如果依然存在说明不是我的设置导致的。

**Reply by heisiheisi** [reply]
用纯净模式看了4天   暂时没出现全屏卡死的问题.  我再试试不用纯净模式  添加参数d3d11-exclusive-fs=no

**Reply by hooke007** [reply]
这参数我本来就没开

**Reply by hooke007** [reply]
你要粗暴排查的话那就分段删配置，看是哪一块引发的问题。

**Comment by ghost** [comment]


**Reply by heisiheisi** [reply]
感谢指导   试过了 不行. 
感觉不是驱动问题,  毕竟用potplayer硬解 软解都用了这么久都没出过问题,  用mpv纯净模式测了几天也没问题.

**Comment by lyturn** [comment]
我这不是偶尔是一点全屏就卡死，mpv一点全屏就卡死，  mpv.net试两三下就卡死。potplayer没有问题。

**Reply by hooke007** [reply]
按照二楼的说明自行排查
